core/modules/system/system.install

```diff
@@ -3123,6 +3123,16 @@ function system_update_1072() {
   // Empty update to force a rebuild of hook_library() and JS aggregates.
 }
 
+/**
+ * Rename system settings to avoid use of 'whitelist'.
+ */
+function system_update_1073() {
+  $allowlist = settings_get('sanitize_input_whitelist', array());
+  global $settings;
+  unset($settings['sanitize_input_whitelist']);
```

Comment:
Is this the right way to delete a setting?

Reply:
Since $settings are in settings.php I don't think this will do anything.